### DISCUSSION TITLE
CP disconnects the connection to the server

### DISCUSSION THREAD

**Discussion by Kn0ck3dL00s3** [Courseplay/Courseplay_FS22#3360] [Q&A]
Hello everyone,

We have a small problem. Two of us play on our server via a router, which Giant seems to have a problem with (my wife takes ages to get onto the server). In addition, when she generates a job with CP, she is disconnected from the server as soon as she presses start. We have tested everything from changing the version of CP to VPN. Unfortunately, none of this has worked. I still have to start my wife's jobs otherwise it won't work.

Does anyone have any ideas?

Thx alot

[log.txt](https://github.com/user-attachments/files/16737194/log.txt)

**Comment by Tensuko** [comment]
2024-08-24 10:15 Error: Running LUA method 'packetReceived'.
C:/Users/Susanne/OneDrive/Dokumente/My Games/FarmingSimulator2022/mods/FS22_Courseplay/scripts/ai/jobs/CpAIJob.lua:441: attempt to index local 'vehicle' (a nil value)

We can have a look at this Error, but not sure if this will solve your issue.

**Reply by Kn0ck3dL00s3** [reply]
Nothing else was to be found in the log? Strange? The fact is that it is perfectly related to the problem that my wife needs 5-10 attempts to access our server just because we play via our router. The support of our server provider once dropped something like that.

Thank you very much for your help and effort.

**Reply by Tensuko** [reply]
If your router sux you buy a new one....
btw everyone connects to the internet via a router....
So you host your game localy or have a dedicated server that you rent?

**Reply by Kn0ck3dL00s3** [reply]
I realise that everyone connects to a router, I just wanted to let you know that we have the same provider IP. And yes, I have a rented dedicated server at Dawn-Server.

**Reply by Kn0ck3dL00s3** [reply]
I just find it strange. We play with 4 people on the server and we all have no problems with CP. Only my wife just can't give any tasks. So I don't think it's because of the Github version.

**Comment by Kn0ck3dL00s3** [comment]
And?

**Reply by Tensuko** [reply]
3/4 ppl work just fine, tell me why CP is the problem then?
That small Error we found in the log is fixed in the next version, anything more network related is not in our hands.
It is else a Giatns Problem or your network.

Nothing we can do about that.
